co-authors-plus.php

```diff
@@ -385,7 +385,9 @@ public function coauthors_meta_box( $post ) {
 		?>
 
 		<div id="coauthors-edit" class="hide-if-no-js">
-			<p><?php _e( 'Click on an author to change them. Drag to change their order. Click on <strong>Remove</strong> to remove them.', 'co-authors-plus' ); ?></p>
+			<?php add_thickbox(); ?>
+			<a class="button thickbox" href="<?php echo get_bloginfo('url'); ?>/wp-admin/users.php?&page=view-guest-authorsTB_iframe=true&width=1200&height=640"><small><?php _e( 'Guest Authors', 'co-authors-plus' ); ?></small></a>
```

Comment:
lets nix thickbox but perhaps hyperlink to guest authors page instead? 

Currently the UX requires producers to click away from the page to add guests. 

Longterm would make more sense to think about a toggle of some kind (thickbox is obviously a functional hack, apologies) to quickly add new guests without clicking away from a post edit screen**. If you've tried the current flow on mobile or tablet it's particularly cumbersome.